test_analysis/Complex_line_shape_fitter.py

```diff
@@ -15,36 +17,57 @@ class ComplexLineShapeTests(unittest.TestCase):
 
     def test_complex_lineshape(self):
         from mermithid.processors.IO import IOCicadaProcessor
-        from mermithid.processors.misc.KrComplexLineShape import KrComplexLineShape
```

Comment:
is the KrComplexLineshape processor not tested anymore? If we are keeping this processor there should also be a test for it.